src/lib/consts.ts

```diff
@@ -0,0 +1 @@
+export const INVALID = "##INVALID";
```

Comment:
```suggestion
export const INVALID_OPENING_HOURS_MARKER = "##INVALID##";
```
et à déplacer dans opening_hours.ts; après c'est tout bon!